README.md

```diff
@@ -65,56 +68,56 @@ This facet is an extension of Governance ERC20, allowing the token to be burned.
 ## Other
 ### Verification
-First plugin to implement [Secure SECO Verification](https://github.com/SecureSECODAO/SecureSECOVerification). Allowing wallets to become members of the DAO by verifying with their 1y old GitHub account or proof of humanity.
+First plugin to implement [Secure SECO Verification](https://github.com/SecureSECODAO/SecureSECOVerification). Allowing wallets to become members of the DAO by verifying with their 1-year-old GitHub account or proof of humanity.
 ### GitHub Pull Request Merger
-Allows the plugin to merge pull request on GitHub. This means that DAO members will be able to maintain the GitHub without needing to trust a centralized team of reviewers. This does however require some setup, which can be found on [their GitHub](https://github.com/SecureSECODAO/SecureSECOPullRequestMerger).
+Allows the plugin to merge pull requests on GitHub. This means that DAO members will be able to maintain the GitHub without needing to trust a centralized team of reviewers. This does however require some setup, which can be found on [their GitHub](https://github.com/SecureSECODAO/SecureSECOPullRequestMerger).
 ## Planned
-This funcionality is planned to be added to the project in the future. These are based on requests seen in the Aragon Discord.
-### Role based membership
-Wallets are granted certain roles and based on this are allowed to use certain functionality of the plugin.
+This functionality is planned to be added to the project in the future. These are based on requests seen in the Aragon Discord.
+### Role-based membership
+Wallets are granted certain roles, and based on this, they are allowed to use certain functionality of the plugin.
 ### NFT governance
 Voting power based on certain ERC721 or ERC1155 tokens present in the wallet.
 ### Dashboard
 Web portal to allow users to easily create a DAO with the Diamond Governance plugin and create proposals to alter the Diamond Governance plugin in existing DAOs.
 
-**This point onwards is meant for developers of the projects, there will be no intresting information for regular users.**
+**This point onward is meant for developers of the projects; there will be no interesting information for regular users.**
 # Setup
 ## Dependencies
-NodeJS (>=14.x.x)  
-npm (>=6.x.x)  
+NodeJS (>=14.x.x)  
+npm (>=6.x.x)  
 
 ## Build process
-npm i  
-npm run compile  
+npm i  
+npm run compile  
 
 ## Secrets
-Copy the .env.example file to a file called .env (located in the same folder). In this file you are required to add the secrets for any of the features you would like to use (MUMBAI_PRIVATE_KEY if you would like to deploy to mumbai for example).
+Copy the .env.example file to a file called .env (located in the same folder). In this file you are required to add the secrets for any of the features you would like to use (MUMBAI_PRIVATE_KEY if you would like to deploy to Mumbai for example).

```

Comment:
I would put stuff like ".env.example" in between some backticks, and optionally add a link to the actual file (can be relative path, e.g. "/.env.example" works in this case (even if intellisense doesn't tell you so, that's because of the dot prefix))
Same for MUMBAI_PRIVATE_KEY (except the adding link to file part ofc)